apcd-cms/src/apps/extension/templates/extension_submission_form/extension_submission_error.html

```diff
@@ -12,7 +12,7 @@ <h1>Request to Submit Extension</h1>
     <p class="c-message c-message--error">
         An error occurred during your extension request submission. For help, <a href="/workbench/dashboard">submit a ticket</a>.
     </p>
-    <a class="c-button c-button--primary" href="/workbench/dashboard">Go to Dashboard</a>
+    <a class="c-button c-button--primary" href="/submissions/extension-request/">Go Back to Extensions/a>
```

Comment:
Error page has a small bug
```suggestion
    <a class="c-button c-button--primary" href="/submissions/extension-request/">Go Back to Extensions</a>
```